fcrepo-mint/src/main/java/org/fcrepo/mint/HttpPidMinter.java

```diff
@@ -184,10 +182,10 @@ public String mintPid() {
             return responseToPid( EntityUtils.toString(resp.getEntity()) );
         } catch ( IOException ex ) {
             LOGGER.warn("Error minting pid from {}: {}", url, ex);
-            throw new RepositoryRuntimeException("Error minting pid", ex);
+            throw new RuntimeException("Error minting pid", ex);
```

Comment:
Can we at least have a semantically-transparent subclass of `RuntimeException`, not just a naked `RuntimeException`? E.g. `PidMinteratingException`?